### DISCUSSION TITLE
Tables and deployments broken after update

### DISCUSSION THREAD

**Discussion by TsengFayt** [cotes2020/jekyll-theme-chirpy#1321] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I started using Chirpy last month and made a ton of tweaks to the css through _sass and some modifications to the basic layout of the sidebar and home pages to get the theme exactly where I wanted it. All was working well until 4 days ago when the new update was pushed. 

Looking back, I think my issue was that since I used chirpy-starter, any of the files I don't explicitly include in my repo, but that Chirpy is looking for, will pull from the master branch on jekyll-theme-chirpy instead of my repo. So somewhere along the lines something is broken. The two major issues are that none of the changes I add to my tables via css are functional now, and when I try to upload a new version and push the changes I get the error:

```
Error: Missing argument $type.
    ┌──> /home/runner/work/garticphonesurvivalguide/garticphonesurvivalguide/_sass/addon/commons.scss
1[15](https://github.com/TsengFayt/garticphonesurvivalguide/actions/runs/6517471129/job/17701976848#step:5:16) │   @include prompt();
    │   ^^^^^^^^^^^^^^^^^ invocation
    ╵
    ┌──> /home/runner/work/garticphonesurvivalguide/garticphonesurvivalguide/_sass/addon/module.scss
180 │ @mixin prompt($type, $fa-content, $fa-style: 'solid') {
    │        ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ declaration
    ╵
  /home/runner/work/garticphonesurvivalguide/garticphonesurvivalguide/_sass/addon/commons.scss 115:3           prompt()
  /home/runner/work/garticphonesurvivalguide/garticphonesurvivalguide/_sass/addon/commons.scss 115:3           @import
  main.scss 7:9                                                                                                @import
  /home/runner/work/garticphonesurvivalguide/garticphonesurvivalguide/assets/css/jekyll-theme-chirpy.scss 1:9  root stylesheet 
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/jekyll-theme-chirpy.scss':
                    Missing argument $type.
                    ------------------------------------------------
      Jekyll 4.3.2   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/home/runner/work/garticphonesurvivalguide/garticphonesurvivalguide/vendor/bundle/ruby/3.2.0/gems/jekyll-sass-converter-3.0.0/lib/jekyll/converters/scss.rb:[17](https://github.com/TsengFayt/garticphonesurvivalguide/actions/runs/6517471129/job/17701976848#step:5:18)5:in `rescue in convert': Missing argument $type. (Jekyll::Converters::Scss::SyntaxError)
```
The only way I was able to recover the site was to re-run an old workflow from a day or so before the update occurred, but the table style still won't apply.

If there is a fix, please let me know. Also, if there a list of files I can add to my repo to prevent it from updating in the future, that would be preferred.

For reference, my repo is at: [Gartic Wiki](https://github.com/TsengFayt/garticphonesurvivalguide)

### Operations you have already tried

- Checked through the most recent commits on jekyll-theme-chirpy to try and find out what files are referenced that might be triggering this new issue. 

- Created a new repo with the most updated version of chirpy-starter to test with my pages (this failed and lead to the index: homepage white-screen-of-death)

- Re-ran workflows from before the update, this fixes the site to at least be live, but the table styles are missing and any new posts cause the site to break when deployed

**Comment by TsengFayt** [comment]
bump

**Comment by joshjohanning** [comment]
If you think you want to make modifications, I would go with the fork method of deploying.

https://chirpy.cotes.page/posts/getting-started/#option-2-github-fork

1. Try to create a fresh instance of the site following that
2. copy in your posts
3. copy in any other assets ie favicons
4. see if your error still persists
